tests/openassetio/hostAPI/test_session.py

```diff
@@ -131,28 +161,49 @@ def test_currentManager(self, a_session, mock_manager_factory, mock_manager_inte
         mock_manager_interface.setSettings.assert_not_called()
         mock_manager_factory.reset_mock()
```

Comment:
[consider][minor] Here and elsewhere, unnecessary `reset_mock`.

Reply:
Think I got them all [here](https://github.com/TheFoundryVisionmongers/OpenAssetIO/pull/214/commits/b0b35e97fb931f0cbba51877d8fea1af39bcca44), and [here](https://github.com/TheFoundryVisionmongers/OpenAssetIO/pull/214/commits/e531d5f4fb13298cc4469ef2ea060c2ca91bd944).